compiler/rustc_session/src/config/cfg.rs

```diff
@@ -470,5 +473,7 @@ impl CheckCfg {
 
         ins!(sym::unix, no_values);
         ins!(sym::windows, no_values);
+
+        ins!(sym::unsigned_char, empty_values).extend(UnsignedCharVar::all());
```

Comment:
the cfg needs to be unstable for now

Reply:
You mean, to hide it behind unstable feature, right?

Reply:
yes, using `cfg(unsigned_char)` should require using an unstable feature

Reply:
`c_char_type` unstable feature added and it is now a feature gated cfg.